package.json

```diff
@@ -22,10 +22,12 @@
   },
   "homepage": "https://github.com/indigotech/onboard-gabriel-moura#readme",
   "dependencies": {
+    "@types/chai-sorted": "0.2.3",
     "@types/jsonwebtoken": "9.0.5",
     "apollo-server": "3.13.0",
     "axios": "1.6.5",
     "bcrypt": "5.1.1",
+    "chai-sorted": "0.2.0",
```

Comment:
essas duas novas dependencias poderiam ser devDependencies em vez de dependencies
https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file